docs/docker/Dockerfile

```
@@ -2,26 +2,36 @@ FROM node:15 as build
 ARG BUILD_MODE
 ENV BUILD_MODE=${BUILD_MODE:-stage}
 
-WORKDIR app
+WORKDIR /app
 
 # dependencies
-COPY package*.json ./
+COPY docs/package*.json ./docs/
+COPY .git ./.git
+RUN pwd
+RUN ls -al
+RUN ls -al docs
+
+# install dependecies
+WORKDIR /app/docs
 RUN yarn
 
 # app
-COPY . ./
+WORKDIR /app
+COPY docs ./docs
+
+WORKDIR /app/docs
 
 # clear non public documentation files
 RUN if [ "$BUILD_MODE" = "production" ] ; then     \
     rm -rf ./next/     \
     rm ./.vuepress/public/handsontable.js ./.vuepress/public/handsontable.css     \
   ; fi
 
-RUN npm run docs:build
+RUN yarn docs:build
```

Can we stick to the same tool that we use while developing the code? 
```suggestion
RUN npm run docs:build
```
and I'd change how the dependencies are installed (line 16) I'd change `RUN yarn` to `RUN npm ci`. The `yarn.lock` is not generated by us so to make sure that docs will be generated from stable dependencies it's a better idea to use `npm`.

---

You are right, done in 1c84363